collections/tenant/Add Certified Attribute.bru

```diff
@@ -0,0 +1,26 @@
+meta {
+  name: Add Certified Attribute
+  type: http
+  seq: 20
+}
+
+post {
+  url: {{host-tenant}}/tenants/:tenantId/attributes/certified
+  body: json
+  auth: none
+}
+
+params:path {
+  tenantId: {{tenantId}}
+}
+
+headers {
+  Authorization: {{JWT}}
+  X-Correlation-Id: {{correlation-id}}
+}
+
+body:json {
+  {
+    "id" : "69e2865e-65ab-4e48-a638-2037a9ee2ee7"
```

Comment:
Could you use a different UUID here, so that it's not equal to the tenant ID (the one you save in the `tenantId` variable) - otherwise one could think this parameter should be the tenant ID, but it's an attribute ID (same for other places where you use this ID as attribute ID)